### ISSUE TITLE
gradle should run some tasks without valid ML credentials

### ISSUE THREAD

**Issue by crism** [marklogic/marklogic-data-hub#1922]
#### The issue

When running `./gradlew tasks` without ML credentials configured, the user gets the message:

```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/jrandom/src/example-ml/DEV-nightly-deployment/build.gradle' line: 7

* What went wrong:
An exception occurred applying plugin request [id: 'com.marklogic.ml-data-hub', version: '4.1.1']
> Failed to apply plugin [id 'com.marklogic.ml-data-hub']
   > No user provided
```

Simple, non-ML-dependent tasks, especially `tasks`, ought to succeed without valid ML credentials.

Secondarily, the error message could be more descriptive: “No valid MarkLogic administrator user/password combo provided,” or something like that. (This message comes up when a user _is_ provided, but is incorrect or does not have the correct password.)

#### Tech details

Which Operating System are you using? CentOS Linux release 7.6.1810

Which version of MarkLogic are you using? 9.0-8

Which version of the Data Hub Framework are you using? 4.1.1

Is this a QuickStart UI bug? No.

#### The devil is in the details...

If possible, rerun the command with -d for debugging output and attach the output:
[gradle.log](https://github.com/marklogic/marklogic-data-hub/files/2903470/gradle.log)

**Comment by clockworked247** [comment]
Thanks for your feedback.  I'm marking this as an enhancement request.

**Comment by dmcassel** [comment]
If this relates to CallResourceTask tasks, see https://github.com/marklogic-community/ml-gradle/issues/558.

**Comment by rjrudin** [comment]
I'm almost certain that's the case. This will be fixed in DHF 5.3.0, so going to close this eagerly.
